src/xcode/ENA/ENA/Source/Scenes/ContactDiary/DiaryCoordinator.swift

```diff
@@ -176,9 +176,9 @@ class DiaryCoordinator {
 		} else {
 			exportString = ""
 		}
-
+		let exportItem = DiaryExportItem(subject: AppStrings.ContactDiary.Overview.ActionSheet.exportActionSubject, body: exportString)
```

Comment:
`AppStrings.ContactDiary.Overview.ActionSheet` is a candidate for a type alias, imho.